wiki/Tournaments/The_Roundtable/16_player_battle_royale/en.md

```diff
@@ -0,0 +1,64 @@
+---
+tags:
+  - The Round table
```

Comment:
```suggestion
  - TRT
```
this is the only tag you need in all of the articles. tags that have all their text contained in the title can be omitted